app/assets/stylesheets/profile.scss

```diff
@@ -113,19 +115,23 @@
   }
 
   #profile {
-    padding: 20px;
     margin-bottom: 35px;
+    border-radius: 20px 20px 0px 0px;
 
     #profile_photo {
       margin-top: 10px;
-      padding-bottom: 10px;
       text-align: center;
     }
 
+    #profile_photo img {
+      border-radius: 20px 20px 0px 0px;
+      border: 1px solid #c1c1c1;
+      border-bottom: 0px;
+    }
+
     ul#profile_information {
-      border-top: 1px dashed $border-grey;
       margin: 0;
-      padding: 0;
+      padding: 0px 10px 0px 10px;
```

Comment:
- Shorthands of length `4` are not allowed. Value was `0px 10px 0px 10px`
- Shorthand form for property `padding` should be written more concisely as `0px 10px` instead of `0px 10px 0px 10px`
- `0px` should be written without units as `0`